Demo/Common/IntegrationExamples/Protocols/AdvancedFlowExampleProtocol.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import AdyenNetworking
+import AdyenSession
+
+internal protocol AdvancedFlowExampleProtocol: AnyObject, APIClientAware {
+    var context: AdyenContext { get }
+    var palApiClient: APIClientProtocol { get }
```

Comment:
let's combine `palApiClient` and `apiClient` into one meta "apiClient" that exposed via `APIClientAware`.

From Demo app and Merchant's perspective - there is no such thing as "palApiClient" - everything will go via merchant's backend.

Reply:
We will discuss this and take this in separate PR.